lib/util/cache/repository/types.ts

```diff
@@ -32,6 +33,13 @@ export interface BranchCache {
   upgrades: BranchUpgradeCache[];
 }
 
+export interface OnboardingBranchCache {
+  isOnboarded: boolean;
+  parentSha?: string;
+  sha: string;
```

Comment:
```suggestion
  defaultBranchSha?: string;
  onboardingBranchSha: string;
```

More easy to understand names.

Also, why is sha mandatory but parentSha optional currently?

Reply:
parentSha is stored in the cache.scan object so I was thinking to use that in future instead of storing the same thing twice.